bucket/qrencode.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": "4.1.1-3",
+    "description": "Encode input data in a QR Code symbol",
+    "homepage": "https://fukuchi.org/works/qrencode/",
+    "license": "LGPL-2.1-only",
+    "architecture": {
+        "64bit": {
+            "url": [
+                "https://mirror.msys2.org/msys/x86_64/libqrencode-4.1.1-3-x86_64.pkg.tar.zst",
+                "https://mirror.msys2.org/msys/x86_64/msys2-runtime-3.3.5-3-x86_64.pkg.tar.zst"
+            ],
+            "hash": [
+                "ead222db63fadf4c6ec67f6b7a95761eb9d0c4ffd92e18129ab11f7944e1e583",
+                "9e99486501905294eb01d7024d2cb96f658b609232878a990e469b5448ea4e11"
+            ]
+        }
+    },
+    "pre_install": [
+        "$files = @('qrencode.exe', 'msys-qrencode-4.dll', 'msys-2.0.dll');",
```

Comment:
We should also keep these files, for the sake of completeness:
```
/usr/share/licenses/libqrencode/COPYING
/usr/share/man/man1/qrencode.1.gz
```

Reply:
Done